toolchain/check/check_unit.h

```diff
@@ -31,6 +31,8 @@ struct PackageImports {
       : package_id(package_id), node_id(node_id) {}
 
   // The identifier of the imported package.
+  //
+  // The current package uses the `IdentifierId::Invalid` id.
```

Comment:
This semantic isn't specific to CheckUnit. It's in check.cpp, PackagingNames, SemIR::File, etc. I don't think it makes sense to start documenting separately in every place. Maybe it'd be better to find a central way to document this? You could probably add an alias if you prefer.

Reply:
To be sure, with an alias, I'm thinking something like a documenting IdentifierId::DefaultPackage or something. But we already have LibraryNameId, so another option would be to add a similar PackageIdentifierId type.